includes/functions.php

```diff
@@ -1579,6 +1587,17 @@ function is_actor( $data ) {
 	 */
 	$types = apply_filters( 'activitypub_actor_types', Actor::TYPES );
 
+	return _is_type_of( $data, $types );
+}
+
+/**
+ * Private helper to check if $data is of a given type set.
+ *
+ * @param array|object|string $data  The data to check.
+ * @param array               $types The types to check against.
+ * @return boolean True if $data is of one of the types, false otherwise.
+ */
+function _is_type_of( $data, $types ) {
```

Comment:
why prefixing?

Reply:
It's WordPress' (imperfect) way to denote a private function that's not part of an API and shouldn't be used outside of Core ([example](https://github.com/WordPress/wordpress-develop/blob/6.8.1/src/wp-includes/cron.php#L1010-L1015)).

Reply:
It's just a helper function. If this was part of a class I'd make it a private method. In the end nothing changes, prefix or not, people can still use it. But at least they know they shouldn't.

Reply:
but why shouldn't they?

Reply:
Because it's a helper function and not part of an API. I think there's a general expectations that functions in this file are fine to be used in third-party plugins. `_is_type_of()` is not part of an API, it's just a helper for the three API functions that use them.

Reply:
Ha! I said the same thing twice 😄